common/objectUtils.js

```diff
@@ -62,8 +60,18 @@ const setInPath = (pathArray = [], value = '') => obj => {
   return obj
 }
 
+const clean = obj => {
+  for (const [key, value] of Object.entries(obj))
+    if (value === null || value === undefined || R.isEmpty(obj))
+      delete obj[key]
+
+  return obj
+}
```

Comment:
why this? The problem should be fixed where the the object is created. don't you think so?

Reply:
it simplifies the code that generates the object itself